MCPForUnity/UnityMcpServer~/src/server.py

```
@@ -101,22 +102,35 @@ def _emit_startup():
-                }
-            )).start()
+            # Initialize connection pool and discover instances
+            _unity_connection_pool = get_unity_connection_pool()
+            instances = _unity_connection_pool.discover_all_instances()
+
+            if instances:
+                logger.info(f"Discovered {len(instances)} Unity instance(s): {[i.id for i in instances]}")
+
+                # Try to connect to default instance
+                try:
+                    default_conn = _unity_connection_pool.get_connection()
```

Ruff (0.14.2)

Local variable `default_conn` is assigned to but never used

Remove assignment to unused variable `default_conn`

(F841)